rpmconf/Install/zmsetup.pl

```diff
@@ -1394,6 +1394,11 @@ sub installLdapConfig {
 
 sub installLdapSchema {
   main::runAsZimbra("/opt/zimbra/libexec/zmldapschema 2>/dev/null");
+  progress ("Updating zimbraLDAPSchemaVersion to version '$ldapSchemaVersion'\n");
```

Comment:
We are already setting the value as default value in zimbra-attrs.xml. 
Can we remove this step completely?

Reply:
My reading is that having specified `requiredIn="globalConfig"` in ldap-attrs.xml, there MUST be a value stored in LDAP, we can't rely on defaulting.